shared/actions/json/fs.json

```diff
@@ -71,6 +71,12 @@
     "uninstallKBFSConfirm": {
       "onSuccess": "() => void"
     },
-    "uninstallKBFS": {}
+    "uninstallKBFS": {},
+    "fsActivity": {},
+    "syncingStatus": {
+      "isSyncing": "boolean"
```

Comment:
As I mention below, if you name this `syncing`, (or the state flag `isSyncing`), you can streamline some of the code a little. And it's cleaner to use the same term everywhere anyway.